todo/old_tests/runtest_textkeys.py

```diff
@@ -58,11 +58,13 @@ def run(self):
             raise ValueError("number of text keys does not match")
         for textkey_import, textkey_export in zip(textkeys_import.text_keys, textkeys_export.text_keys):
             if abs(textkey_import.time - textkey_export.time) > 0.0001:
-                raise ValueError("key times do not match (%f != %f)"
-                                 % (textkey_import.time, textkey_export.time))
+                raise ValueError(f"key times do not match "
```

Comment:
Can be combined into one line, we don't adhere to pep80.

Reply:
Updated. Sorry, 80col is a habit I'll try to avoid for your repo :)

Reply:
No worries, just go with that seems sensible to read.